tests/services/conftest.py

```diff
@@ -44,3 +44,10 @@ def warning_submission_mock(mocker: MockerFixture, validate_submission_mock: Moc
     mock_validation = mocker.patch("sbl_filing_api.services.submission_processor.validate_phases")
     mock_validation.return_value = (False, pd.DataFrame([["warning"]], columns=["validation_severity"]))
     return validate_submission_mock
+
+
+@pytest.fixture(scope="function")
+def df_to_json_mock(mocker: MockerFixture, validate_submission_mock: Mock):
+    mock_json_formatting = mocker.patch("sbl_filing_api.services.submission_processor.df_to_json")
+    mock_json_formatting.return_value = "[{}]"
+    return mock_json_formatting
```

Comment:
Small update based on the latest changes to the data-validator.  The "error" and "warning" strings are capitalized to "Error" and "Warning".  So the mocks in conftest and the submission_processor.py should be updated to conform to that now.  Figured easiest to do part of this PR since you're already in there instead of a whole new story.  You could make it look for case-insensitive if you want to get fancy, otherwise you can just update those strings to start with a capital letter.